README.md

```diff
@@ -39,6 +39,18 @@ installed: /etc/systemd/system/v2ray@.service
 # bash <(curl -L https://raw.githubusercontent.com/v2fly/fhs-install-v2ray/master/install-release.sh)
 ```
 
+### V2Ray 組態
+1. 產生配置文檔 [https://intmainreturn0.com/v2ray-config-gen](https://intmainreturn0.com/v2ray-config-gen/)
+2. 將配置文件 config.json放入 /usr/local/etc/v2ray/config.json
+
+### 離線安裝
+在具有網路下載限制的環境中，我們建議：
+1. 從GitHub.com下載fhs-install-v2ray存儲庫作為zip文件。
+2. 從以下位置下載v2ray-core發行zip文件  [https://github.com/v2fly/v2ray-core/releases](https://github.com/v2fly/v2ray-core/releases)
+3. 將兩個 zip 文檔都上傳到您的伺服器
+4. 解壓縮fhs-install-v2ray存儲庫zip文件
+5. 運行安裝，並將其指向本地v2ray-core zip文件：```sudo bash install-release.sh --local /path/to/v2ray-linux-64.zip```
```

Comment:
1. Chinese and English need spaces 
2. Which files to upload? 
3. ...

I think this is more suitable for a personal blog

Reply:
1. Can you please suggest the spaces? Chinese is not my native language, my apologies
2. Many users are struggling to find documentation on how to set up v2ray with a simple but secure configuration. I would suggest this is something that should be included with the installation.